File: abc295/abc295_c.js

function main(input) {
	const args = input.split('\n');

	const N = parseInt(args[0]);
	const A = args[1].split(' ');

	let ans = 0;
	let dat = {};

	for (let i = 0; i < N; i++) {
		if (A[i] in dat) {
			dat[A[i]]++;
		} else {
			dat[A[i]] = 1;
		}

		if (dat[A[i]] == 2) {
			ans++;
			dat[A[i]] = 0;
		}
	}

	console.log(ans);

}

main(require("fs").readFileSync("/dev/stdin", "utf8"));

File: abc295/abc295_c.py

def main():
    N = int(input())
    A = list(input().split(" "))

    dat = {}
    ans = 0

    for i in range(N):
        dat.setdefault(A[i], 0)

        dat[A[i]] += 1
        if dat[A[i]] == 2:
            ans += 1
            dat[A[i]] = 0

    print(ans)


main()
